tools/cxbe/Xbe.cpp

```diff
@@ -17,6 +17,8 @@
 #include <stdlib.h>
 #include <time.h>
 
+#define MAX(x, y) (((x) > (y)) ? (x) : (y))
+
```

Comment:
I'd prefer to just use `std::max` instead

Reply:
Fair enough. It should be better with the latest commit. 

Feel free to nit style.